src/Install/Installer.php

```diff
@@ -276,6 +279,49 @@ private function awaitingMollieOrderState($languageId)
         return true;
     }
 
+    /**
+     * @param $languageId
+     * @return bool
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
+     */
+    public function orderCompletedOrderState($languageId)
```

Comment:
hard to understand when a function contains order and then order again :D maybe we can replace with different name ?

Reply:
all mollie status creation functions ends with OrderState so i wanted to leave them all similar to each other. Should we change it anyway? 
example : partialRefundOrderState, awaitingMollieOrderState, partialShippedOrderState, orderCompletedOrderState

Reply:
lets keep it